sensor_array/analysis/array_analysis_SVD.py

```diff
@@ -93,13 +93,9 @@ def unify_henrys_data_and_average_constants(dict_of_data_frames):
 
     return df_unified
 
-    
-def calculate_all_arrays_list(mof_list, num_mofs):
-    mof_array_list = []
-    array_size = min(num_mofs)
-    while array_size <= max(num_mofs):
-        mof_array_list.extend(list(combinations(mof_list, array_size)))
-        array_size += 1
+
+def calculate_all_arrays(mof_list, array_size):
+    mof_array_list = list(itertools.combinations(mof_list, array_size))
```

Comment:
list()?

Reply:
List is necessary to return a second sorted list after evaluation.